homework_4/bets_api/internal/api/controllers/controller.go

```diff
@@ -0,0 +1,90 @@
+package controllers
+
+import (
+	"github.com/gin-gonic/gin"
+	"net/http"
+)
+
+type Controller struct {
+	idValidator     IdValidator
+	statusValidator StatusValidator
+	betService      BetService
+}
+
+func NewController(idValidator IdValidator, statusValidator StatusValidator, betService BetService) *Controller {
+	return &Controller{
+		idValidator:     idValidator,
+		statusValidator: statusValidator,
+		betService:      betService,
+	}
+}
+
+func (b *Controller) FetchBetById() gin.HandlerFunc {
```

Comment:
Why Fetch?
I suggest having a consistent name with action names.
`GetBetById` would possibly be a better name here

Reply:
Same with other methods